remoteit/desktop
# DESK-819 service in both accounts

### Description


Fixed device list selection errors when another user's device list is active and you edit the local device configuration.

### Test plan



1. Sign in
2. Select another accounts device list
3. Go to the settings page and click to configure the local device
4. You should be able to add/remove services and change all config options as normal

### Ticket



<https://remoteit.atlassian.net/browse/DESK-819>
